src/client/src/containers/ReviewAndGenerate/index.tsx

```diff
@@ -123,7 +107,6 @@ const mapDispatchToProps = (
 });
 
 const mapStateToProps = (state: AppState): IStateProps => ({
-  projectTypeRows: WizardSelectors.getProjectTypeRowItemSelector(state),
```

Comment:
Since this selector won't be used during our scope, maybe it will be good in the WizardSelector.ts to make a comment or comment out the section of code related to this selector, but this is just a suggestion

Reply:
I don't think this is a big deal since I decided to remove the minimum in case we decide to add components or selectors back in later. I could also make a PR where we completely clean up everything later once we decide